src/index.js

```diff
@@ -20,3 +20,10 @@ addTodoBtn.addEventListener('click', () => {
     render(todosList);
   }
 });
+
+// clear all completed todos
+const clearBtn = document.querySelector('.clear-btn');
```

Comment:
- [ ] According to Project requirements you should create a new JavaScript file to contain methods related to the status updates (completed: true / false).and import it as a module:
- Kindly make this. 👎🏻